src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -37,49 +36,40 @@ export type ActionName =
 
 type ActionTypes = {
     actionName: ActionName;
-    newStatus: StatusType;
```

Comment:
There was no reason to have this set out here, and now that Generate Map doesn't save a status change it makes more sense for each modal to know about its own status behaviour